### PR TITLE
SAK-32266 Added bullhorns table to conversions scripts

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: reference/docs/conversion/sakai_12_oracle_conversion.sql [hunk 1]
```diff
@@ -372,3 +372,31 @@ ALTER TABLE SAM_QUESTIONPOOL_T MODIFY DESCRIPTION CLOB;
...
+    SITE_ID VARCHAR2(99),
+    URL CLOB NOT NULL,
+    EVENT_DATE TIMESTAMP NOT NULL,
+    PRIMARY KEY(ID)
+);
+
+CREATE SEQUENCE bullhorn_alerts_seq;
+
+BEGIN
+EXECUTE IMMEDIATE
```
Comment (juanjmerono) [inline reference/docs/conversion/sakai_12_oracle_conversion.sql]: You don't need to do this little trick just add the real sentence:
```
CREATE OR REPLACE TRIGGER bullhorn_alerts_bir
 BEFORE INSERT ON BULLHORN_ALERTS
 FOR EACH ROW
 BEGIN
      SELECT bullhorn_alerts_seq.NEXTVAL
      INTO   :new.id
      FROM   dual; 
END;
/
```

**Comment by adrianfish** [comment]
Okay, thanks. I've updated it.

File: reference/docs/conversion/sakai_12_oracle_conversion.sql (hunk 1 above)
Comment (bjones86) [reply reference/docs/conversion/sakai_12_oracle_conversion.sql]: I noticed that the trailing forward slash after `END;` didn't make it into the conversion script. Is this bit necessary? We ran this as is (without the slash), and it ended up creating the trigger in an invalid format, and thus wouldn't compile, leading to stack traces. When I inspected the trigger, this is how it ended up being created:

```
create or replace TRIGGER bullhorn_alerts_bir
    BEFORE INSERT ON BULLHORN_ALERTS
    FOR EACH ROW
    BEGIN
        SELECT bullhorn_alerts_seq.NEXTVAL
        INTO   :new.id
        FROM   dual
```

Notice the absence of the semicolon after `FROM dual`, and the missing `END;` on the following line. Could this be explained by the missing forward slash in comment above?

cc: @adrianfish, @jmarianolujan

Comment (bjones86) [reply reference/docs/conversion/sakai_12_oracle_conversion.sql]: I also found this article: https://stackoverflow.com/questions/1079949/when-do-i-need-to-use-a-semicolon-vs-a-slash-in-oracle-sql, which states:

> The / is mainly required in order to run statements that have embedded ; like a CREATE PROCEDURE statement.

Comment (bjones86) [reply reference/docs/conversion/sakai_12_oracle_conversion.sql]: We've confirmed locally that the trailing slash is necessary. Without it, the script fails.

**Pull request merged**